common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,387 @@
+	Your browser does not support the video tag.
+</video>
+<pre>
+<code>&lt;video controls&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video1-en.webm" type="video/webm"&gt;
+Your browser does not support the video tag.
+&lt;/video&gt;</code>
+</pre>
+
+<h2>The <code>audio</code> element</h2>
+<audio controls>
+	<source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.mp3" type="audio/mp3"/>
+	<source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.ogg" type="audio/ogg"/>
+	Your browser does not support the audio tag.
+</audio>
+<pre>
+<code>&lt;audio controls&gt;
+&lt;source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.mp3" type="audio/mp3"/&gt;
+&lt;source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.ogg" type="audio/ogg"/&gt;
+Your browser does not support the audio tag.
+&lt;/audio&gt;</code>
+</pre>
+
+<h2>The <code>track</code> element</h2>
+<video controls>
+	<source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.webm" type="video/webm">
+	<source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.mp4" type="video/mp4">
+	<track src="demo/video2-captions-en.xml" kind="captions" data-type="application/ttml+xml" srclang="en" label="English">
+	Your browser does not support the video tag.
+</video>
+<pre>
+<code>&lt;video controls&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.webm" type="video/webm"&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.mp4" type="video/mp4"&gt;
+&lt;track src="demo/video2-captions-en.xml" kind="captions" data-type="application/ttml+xml" srclang="en" label="English"&gt;
+Your browser does not support the video tag.
+&lt;/video&gt;</code>
+</pre>
+
+<h2>The <code>map</code> and <code>area</code> elements</h2>
+
+<h2>The <code>details</code> and <code>summary</code> element</h2>
+
+<details>
+<summary>Summary</summary>
+<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus sagittis metus ligula, aliquet vulputate ligula molestie nec. Aenean dapibus varius dapibus.</p>
+</details>
+<pre>
+<code>&lt;details&gt;
+&lt;summary&gt;Summary&lt;/summary&gt;
+&lt;p&gt;Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus sagittis metus ligula, aliquet vulputate ligula molestie nec. Aenean dapibus varius dapibus.&lt;/p&gt;
+&lt;/details&gt;</code>
+</pre>
+
+<h2>The <code>dialog</code> element</h2>
+
+<dialog open>Lorem ipsum dolor sit amet, consectetur adipiscing elit.</dialog>
+<pre>
+<code>&lt;dialog open&gt;
+Lorem ipsum dolor sit amet, consectetur adipiscing elit.
+&lt;/dialog&gt;</code>
+</pre>
+
+<h2>The <code>table</code> element</h2>
+
+<table></table>
+<pre>
+<code>&lt;table&gt;&lt;/table&gt;</code>
+</pre>
+
+<h3>2 rows & 2 columns with the <code>tr</code> and <code>td</code> elements</h3>
+
+<table>
```

Comment:
The table should be in its own section, sibling to "Embedded content" and "form". Please create a new page for those example in the scaffolding.

Reply:
Fixed in commit "Addressed changes".